src/SFML/Window/GlContext.cpp

```diff
@@ -199,6 +200,26 @@ namespace
 
     // Supported OpenGL extensions
     std::vector<std::string> extensions;
+
+    // Helper to parse OpenGL version strings
+    bool parseVersionString(const char* version, const char* prefix, unsigned int &major, unsigned int &minor)
+    {
+        std::size_t versionPosition = std::strlen(prefix);
+
+        if ((std::strlen(version) >= (versionPosition + 3)) &&
+            !std::strncmp(version, prefix, std::strlen(prefix)) &&
```

Comment:
I guess the third parameter could be replaced with `versionPosition`, but this is a minor detail.